x-pack/plugins/observability_solution/observability_ai_assistant/server/functions/summarize.ts

```diff
@@ -62,21 +63,31 @@ export function registerSummarizationFunction({
         ],
       },
     },
-    (
-      { arguments: { id, text, is_correction: isCorrection, confidence, public: isPublic } },
+    async (
+      { arguments: { id: docId, text, is_correction: isCorrection, confidence, public: isPublic } },
       signal
     ) => {
+      // The LLM should be able to update an existing entry by providing the same doc_id
+      // if no existing entry is found, we generate a uuid
+      const id = await client.getUuidFromDocId(docId);
```

Comment:
This makes it possible for the LLM to blindly suggest a doc_id, without any information about existing entries, and with that we can retrieve the `_id`. It does work but I don't like it very much. 

A better approach might be to get rid of `doc_id` entirely. We already provide the LLM with relevant entries via recall. By improving the recall to also include contradicting entries (which I've done in this PR) the LLM should be able to get the `_id` for the existing entry and use that in order to update it.